include/circt/Dialect/ESI/ESIPorts.td

```diff
@@ -57,7 +57,7 @@ def WrapValidReady : ESI_Op<"wrap.vr", [NoSideEffect]> {
   ];
 }
 
-def UnwrapValidReady : ESI_Op<"unwrap.vr", [NoSideEffect]> {
```

Comment:
I'm not at all up to speed on ESI but I'm curious why this sort of operation has side effects?

Reply:
Whoops. I meant to make `unwrap.iface` side-effectful. I can't find a definition of `NoSideEffect` so I'm interpreting it as "has no effect on anything besides its results". `unwrap.iface` outputs signals on the `!sv.modport` is takes as an argument. Is that correct?

Reply:
Right.  `NoSideEffect` is the equivalent of LLVM's ["readnone" attribute](http://llvm.org/docs/LangRef.html#fnattrs).